src/app/page/blog/blog.component.html

```diff
@@ -18,8 +18,12 @@
 
 <div *ngIf="feed" class="main-div">
   <ng-container *ngFor="let item of feed">
-    <div class="feed-card">
-        <div class="title">{{ item.title }}</div>
+    <div [ngClass]="{
+                 'feed-card': item.pin_unpin== 0, 
```

Comment:
You may have to update this based on renamed value in backend. (`pin_unpin` -> `pinned`)